rules/CodeQuality/NodeAnalyzer/ConstructorPropertyDefaultExprResolver.php

```diff
@@ -33,6 +35,10 @@ public function resolve(ClassMethod $classMethod): array
         $defaultPropertyExprAssigns = [];
 
         foreach ($stmts as $stmt) {
+            if ($stmt instanceof If_ && $this->hasReturnStmt($stmt)) {
+                break;
```

Comment:
I think better to use "break;" when $stmt is not Expression, so handle in next code to use break; instead of continue.

Reply:
OK, simplier than i thought, thx.